### PR TITLE
Shifts PQ Requirements for ALL Roles

### PR DESCRIPTION
**Pull request by SaintBurgher**
## About The Pull Request
This PR is Guayo and Saint (obviously) approved. It focuses on keeping the restrictions light for those who have good behavior but also punishing for those who go down into the negatives.

## Why It's Good For The Game
Allows more restrictions for people to take negative PQ seriously.

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/serfs/blacksmith.dm [hunk 1]
```diff
@@ -14,6 +14,7 @@
 	display_order = JDO_ARMORER
 	bypass_lastclass = TRUE
 	give_bank_account = 30
+	min_pq = -10
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/serfs/blacksmith.dm:17]: As a smith player I suggest leaving this at a minimum of zero. Shitters playing a smith are a danger....

Comment (ThePainkiller) [reply code/modules/jobs/job_types/roguetown/serfs/blacksmith.dm]: Ohhh good call on that one, I like it

File: code/modules/jobs/job_types/roguetown/garrison/garrisonguard.dm [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 	outfit = /datum/outfit/job/roguetown/guardsman
 	give_bank_account = 30
-	min_pq = -4
+	min_pq = -50
```
Comment (lavillastrangiato) [inline code/modules/jobs/job_types/roguetown/garrison/garrisonguard.dm:27]: Why does this have a lower PQ requirement than the Squire?

**Review by Optimism333 — COMMENTED** [review]
Don't put garrison PQ requirements at -50. People who get nuked should be locked off combat roles, particularly ones that have authority/responsibility in the town. The value of -4 from before was fine.

**Comment by 1N7Slayer** [comment]
Yeah, some of these are a bit wacky. Going below -5 for whatever reason isn't normal player conduct, a person should be rewarded for their behavior to at least be able to play fantasy shitcurity.

**Comment by SaintBurgher** [comment]
I brought Garrison up to 0 to be consistent with squire.

**Comment by SaintBurgher** [comment]
Cleared the conflicts here

**Review by Stutternov — APPROVED** [review]
Maintainer aboose

**Pull request merged**